README.md

```diff
@@ -1,150 +1,53 @@
-# 1Password Connect Node SDK
+<!-- Image sourced from https://blog.1password.com/introducing-secrets-automation/ -->
+<img alt="" role="img" src="assets/connect-illustration.svg"/>
```

Comment:
I'm wondering, is there a way to provide a link to the image here? Similar to what we do for the Get started button.
I ask this so that we don't have to re-upload the illustration in each repo.

Reply:
Great callout! The only risk with not having a local copy of the image in the repo is if the remote resource ever becomes unavailable.

My usual instinct is to make any many assets locally available as possible, but since these images are just decorative and don't convey any content or meaning about the repo itself, I agree that linking to them is the better approach. Even in the event that the remote resource is unavailable, the image just won't render, but nothing will be broken or lost as a result of that.

Will update shortly 👍

Reply:
Addressed via 283e2b205e208b0ca2c1320e89e5426bc9acb665 commit.